tests/test_treeherder_menu.py

```diff
@@ -13,12 +13,12 @@ def test_treeherder_dropdown(base_url, selenium):
     treeherder_page = TreeherderPage(selenium, base_url).open()
 
     # Verify Treeherder loads
-    assert treeherder_page.unclassified_failure_count > 0
+    assert treeherder_page.result_sets > 0
```

Comment:
This is effectively identical to to the `wait_for_page_to_load` in `TreeherderPage`, which is implicitly called by the `open` method. I think we can safely remove this assertion and the comment.